en/docs/develop/components/api-proxies/policies.md

```diff
@@ -0,0 +1,88 @@
+# Policies
+
+Policies are units of business logic that a user can apply to slightly change the  `Request`, `Response`, or `Error` flow of an API invocation before reaching the backend or the client. For example, you can modify the response of an API invocation by adding a policy to the Response flow to transform the payload from JSON to  XML and to add a header in the response flow. 
```

Comment:
Was "to add a header in the response flow." meant to be "to add a header in the response."?

Reply:
done.